.github/workflows/diff-test-suite-agent.yml

```
@@ -0,0 +1,49 @@
+name: Diff Test Suite Agent
+
+on:
+  pull_request:
+    branches: [main, develop]
+
+jobs:
+  generate-tests:
+    runs-on: ubuntu-latest
+    if: ${{ secrets.QODO_API_KEY != '' }}
```

actionlint (1.7.8)

context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)